addon/components/layers-dialogs/edit.js

```diff
@@ -504,6 +562,51 @@ let FlexberryEditLayerDialogComponent = Ember.Component.extend(
         Ember.set(layer, 'coordinateReferenceSystem', coordinateReferenceSystem);
 
         let settings = Ember.get(layer, 'settings');
+
+        let boundsMode = this.get('boundsMode');
+        let geoJsonBounds;
+
+        // Coordinates should be projected to LatLngs.
+        if (boundsMode === 'bbox') {
+          let bbox = Ember.get(layer, 'settings.bbox');
+
+          if (!Ember.isBlank(bbox[0][0]) && !Ember.isBlank(bbox[0][1]) &&
+            !Ember.isBlank(bbox[1][0]) && !Ember.isBlank(bbox[1][1])) {
+
+            // Put it in property to force LeafletCRSMixin do it's work.
+            this.set('coordinateReferenceSystem', coordinateReferenceSystem);
```

Comment:
property is unnecessary here.
you should create and export function in crs mixin and use it here.